src/Models/SiteTreeLink.php

```diff
@@ -134,4 +134,19 @@ public function getTitle(): ?string
         // Use page title as a default value in case CMS user didn't provide the title
         return $page->Title;
     }
+
+    public function getLinkTypeDefaultTitle(): string
+    {
+        $pageExist = $this->Page()->exists() && $this->getURL();
```

Comment:
```suggestion
        $page = $this->Page();
        $pageExist = $page->exists();
```
We need this variable to grab the title for the return at the end of the method.

Reply:
Updated